docs/support/faq/get-image-format.md

```diff
@@ -0,0 +1,115 @@
+# Why Rahti cannot find this docker image?
+
+![Could not load image](img/Could_not_load_image_metadata.png)
+
+There are simple causes to this problem. Maybe there is a typo in the image name, or maybe the image has been deleted since the last time it was successfully pulled. These two problems are common, and it is worth to double check the source.
+
+![Failed to pull image](img/Failed_to_pull_image.png)
+
+## Private
+
+Other option is that maybe the image is private. In this case it is necessary to setup a `docker-registry` secret with an account that has permissions to pull the image. For example for an image stored in docker hub:
+
+```bash
+$ oc create secret docker-registry myprivaterepoaccess \
+      --docker-username=<USERNAME> \
+      --docker-server=docker.io \
+      --docker-email=<EMAIL> \
+      --docker-password=<PASSWORD>
+secret/myprivaterepoaccess created
+
+$ oc secrets link default myprivaterepoaccess --for=pull
+```
+
+**Note**: Substitute place holders with actual username, password and email (without <>).
+secret-name.
+
+You can find more information in the [How to add docker hub credentials to a project](../../../cloud/rahti/tutorials/docker_hub_login/) article.
+
+## Unsupported image format
+
+Other more obscure problem is when the format of the image is not supported by the current version of Rahti (v3.11), which uses an old version of the docker client. Currently there are two docker image formats, docker (`application/vnd.docker.container.image.v1+json`) and OCI (`application/vnd.oci.image.manifest.v1+json`), the current version of Rahti only supports `docker`.
+
+When an old client is used to try to pull a image with the newer format, the client cannot find it and returns a `repository does not exist` error. There is not a trivial and definitive procedure to confirm that the image indeed exists but it is stored in the newer format. One way is to use an old version of docker (Rahti uses the one coming with centos7) to try to pull the image, and then use a newer version to try to pull the same image. If it pulls in the new, but not in the old, it is a clear indication that the image is using the new format.
+
+### Script
+
+Other option is to use the script below. When an image is pulled from a registry, the REST API of the registry is used. In the REST request it is necessary to indicate the accepted media type using an HTTPD header. The script makes a request using the old format media type. If the return code is `200` the image is in the old format, and if it is `404` the new.
```

Comment:
```suggestion
An alternative option to determine the format of an image is to use the script below. When an image is pulled from a registry, the REST API of the registry is used. In the REST request, it is necessary to indicate the accepted media type using an HTTP header. The following script makes a request using the old format media type. If the return code is `200` the image is in the old format, and if it is `404` the new.
```

Reply:
what about just using `docker manifest inspect <image>:<tag>` instead of a script?

Reply:
I did not know this existed :/